web-components/src/components/inputs/new/LocationField/LocationField.utils.ts

```diff
@@ -0,0 +1,15 @@
+export const getLongLat = (value: string) => {
+  const isCoordinates =
```

Comment:
It might be useful to extract the regular expression into a constant and move the validation into a function to make the code a bit easier to read. Something like this:
```
const COORDINATES_REGEX = /^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?),\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$/;

const isValidGeoCoordinates = (value: string): boolean => COORDINATES_REGEX.test(value);

export const getLongLat = (value: string) => {
    ...
    if(isValidGeoCoordinates(value)) {
       ...
    }
    ...
};
```

Reply:
updated, please have another look, thanks!